crates/jarscanner/src/main.rs

```diff
@@ -0,0 +1,47 @@
+use clap::Parser;
+use std::collections::HashMap;
+use std::error::Error;
+use std::path::PathBuf;
+
+use bzl_gen_jarscanner as jarscanner;
+
+#[derive(Parser)]
+#[command(author, version, about)]
+#[command(version = "0.1.0")]
+#[command(about = "Jar to containing classes tool", long_about = None)]
+struct Opt {
+    #[arg(long)]
+    out: String,
+
+    #[arg(long)]
+    label: String,
+
+    #[arg(long)]
+    input_jar: String,
```

Comment:
I think you can use `PathBuf` here or something directly so we don't need to convert from String. Same with the `out` above I think and the relative path below.